ctype.md

```diff
@@ -0,0 +1,103 @@
+# 💠 CTYPE  
+
+Before the <span class="label-role claimer">claimer</span> can make a claim about themselves, they first need a CTYPE.  
+
+CTYPE is a KILT-specific term, but the concept is simple.  
+A CTYPE defines the structure of a claim. You can think of it as the data model for your claim.   
```

Comment:
"A CTYPE is a KILT-specific term, but the concept is simple." 

Needs an 'A' at the start.
Nice :)